staff/abdou-bousiyd/precourse/pasapalabra_html/main.js

```diff
@@ -0,0 +1,101 @@
+
+//welcome()
+
+let position = 0;
+let repited = false;
+let time = 4;
+let name = null;
+
+
+
+function draw() {
+    questions.forEach((q, i) => {
+        let rotate = (360 / questions.length) * i + (-90);
+        let li = document.createElement("li");
+        li.appendChild(document.createTextNode(q.letter));
+        li.style.transform = `rotate(${rotate}deg) translate(18em) rotate(${rotate * (-1)}deg)`;
+        document.getElementById('list').appendChild(li);
+    });
+
+}
+draw()
```

Comment:
It's better to execute this function after all functions declarations. As it improves readability of your code. But well done on the Passapalabra game. It's pretty neat